.pre-commit-config.yaml

```diff
@@ -50,7 +50,7 @@ repos:
     files: "^(src/)"
 
 - repo: https://github.com/ansys/pre-commit-hooks
-  rev: v0.3.0
+  rev: v0.2.9
```

Comment:
I think this probably wants to be 0.3.0, rather than the downgrade?

Reply:
(I have no idea why pre-commit.ci is trying to downgrade it, it did this with recordlists)

Reply:
I just didn't cherry-pick the latest pre-commit PR in. main branch is up to date